frontend/src/scenes/saved-insights/SavedInsightsFilters.tsx

```diff
@@ -0,0 +1,99 @@
+import { LemonSelect } from 'lib/components/LemonSelect'
+import { DateFilter } from 'lib/components/DateFilter/DateFilter'
+import { CalendarOutlined } from '@ant-design/icons'
+import { SavedInsightsTabs } from '~/types'
+import { INSIGHT_TYPE_OPTIONS } from 'scenes/saved-insights/SavedInsights'
+import { useActions, useValues } from 'kea'
+import { dashboardsModel } from '~/models/dashboardsModel'
+import { savedInsightsLogic } from 'scenes/saved-insights/savedInsightsLogic'
+import { membersLogic } from 'scenes/organization/Settings/membersLogic'
+import { LemonInput } from 'lib/components/LemonInput/LemonInput'
+
+export function SavedInsightsFilters(): JSX.Element {
+    const { nameSortedDashboards } = useValues(dashboardsModel)
+    const { setSavedInsightsFilters } = useActions(savedInsightsLogic)
+    const { filters } = useValues(savedInsightsLogic)
+
+    const { tab, createdBy, insightType, dateFrom, dateTo, dashboardId, search } = filters
+
+    const { meFirstMembers } = useValues(membersLogic)
+
+    return (
+        <div className="flex justify-between gap-2 mb-2 items-center flex-wrap">
+            <LemonInput
+                type="search"
+                placeholder="Search for insights"
+                onChange={(value) => setSavedInsightsFilters({ search: value })}
+                value={search || ''}
+            />
+            <div className="flex items-center gap-2 flex-wrap">
+                <div className="flex items-center gap-2">
```

Comment:
ah, the move to a new file hides this...

The change in this file is to add this block which lets you select a dashboard to filter the list by. _Really_ it should be a `LemonMultipleSelect` but the ANT component that wrapped hated being in this row so I thought I'd get this in without the rabbit hole